src/main/java/nextstep/ladder/entity/Ladder.java

```diff
@@ -0,0 +1,39 @@
+package nextstep.ladder.entity;
+
+import static nextstep.ladder.exception.UserExceptionMesssage.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import nextstep.ladder.exception.UserException;
+
+public class Ladder {
+	private final List<Line> lines;
+	private final Users users;
+
+	public Ladder(Users users, int ladderHeight) {
+		validateLadderHeight(ladderHeight);
+
+		this.lines = IntStream.range(0, ladderHeight)
+			.mapToObj((i) -> new Line(users.getUsers().size()))
+			.collect(Collectors.toList());
+		this.users = users;
+	}
+
+	private void validateLadderHeight(int ladderHeight) {
+		if (ladderHeight < 0) {
+			throw new UserException(NEGATIVE_NOT_AGRRESIVE);
+		}
+	}
+
+	@Override
+	public String toString() {
+		String userPrefix = users.toString();
+		String ladder = this.lines.stream()
+			.map(line -> String.format("    %s \n", line.toString()))
+			.collect(Collectors.joining());
+
+		return userPrefix + "\n" + ladder;
```

Comment:
OS 마다 줄바꿈을 인식하는게 다르기 때문에 jdk 1.7부터 도입된 현재 OS에 맞는 줄바꿈을 얻어오도록 하면 좋을 것 같아요!
```suggestion
		return userPrefix + System.lineSeperator() + ladder;
```

Reply:
아 글쿤여! 감사합니다 ㅎㅎ